### PR TITLE
shutdown command fixes

### PR DESCRIPTION
**Pull request by lnussel**
Fixes embarrassing c&p error in the shutdown --show implementation. Mea culpa. :face_with_spiral_eyes:

### CODE REVIEW

File: src/systemctl/systemctl-logind.c [hunk 1]
```diff
@@ -339,9 +330,7 @@ int logind_schedule_shutdown(void) {
                 return log_warning_errno(r, "Failed to call ScheduleShutdown in logind, proceeding with immediate shutdown: %s", bus_error_message(&error, r));
 
         if (!arg_quiet)
-                log_info("%s scheduled for %s, use 'shutdown -c' to cancel.",
-                         log_action,
-                         FORMAT_TIMESTAMP_STYLE(arg_when, arg_timestamp_style));
+                logind_show_shutdown();
```
Comment (yuwata) [inline src/systemctl/systemctl-logind.c:336]: I do not think it is not necessary to retrieve the `ScheduledShutdown` property here.
How about to introduce tiny helper `log_scheduled_shutdown()` instead?

**Review by yuwata — CHANGES_REQUESTED** [review]

File: src/systemctl/systemctl-logind.c [hunk 2]
```diff
@@ -308,24 +307,16 @@ int logind_schedule_shutdown(void) {
...
                 action = "poweroff";
-                log_action = "Shutdown";
                 break;
         case ACTION_KEXEC:
                 action = "kexec";
-                log_action = "Reboot via kexec";
-                break;
-        case ACTION_EXIT:
-                action = "exit";
-                log_action = "Shutdown";
```
Comment (yuwata) [inline src/systemctl/systemctl-logind.c:323]: Maybe, it is better to drop `ACTION_EXIT` in a separate commit.

File: src/systemctl/systemctl-logind.c (hunk 1 above)
Comment (lnussel) [reply src/systemctl/systemctl-logind.c:336]: reading back the property makes sure we actually print the server side view though.

File: src/systemctl/systemctl-logind.c (hunk 2 above)
Comment (lnussel) [reply src/systemctl/systemctl-logind.c:323]: Right. I'll put it back in for now instead. There's more cleanup to do in that area

File: src/systemctl/systemctl-logind.c (hunk 1 above)
Comment (keszybz) [reply src/systemctl/systemctl-logind.c:336]: Yeah, I think this makes things simpler and easier.

**Pull request merged**